content/2016-04-05-benchmarks.rst

```diff
@@ -0,0 +1,185 @@
+Compiler Flags
+##############
+
+:date: 2016-04-05
+:category: benchmark
+:lang: en
+:authors: Lightjohn
+:summary: Let's re-run an existing benchmark with pythran 
+
+The benchmark
+=================
+
+First there was a paper [0]_, 
+in the paper there was a github [1]_, 
+in the github [1]_ some benchmarks. 
+In my case I wanted to re-run the julia code because the language is changing quickly and so may do better. 
+But the day before I discovered `Pythran` so why not test both ?
+
+And so let's re-run two benchmarks: C++, Julia and add a new one, Pythran.
+
+Compiling and running C++ code was easy:
+
+	g++ -Ofast RBC_CPP.cpp -o testcpp
+
+
+then
+
+	testcpp 
+	Output = 0.562731, Capital = 0.178198, Consumption = 0.384533
+	Iteration = 1, Sup Diff = 0.0527416
+	Iteration = 10, Sup Diff = 0.0313469
+	Iteration = 20, Sup Diff = 0.0187035
+	Iteration = 30, Sup Diff = 0.0111655
+	Iteration = 40, Sup Diff = 0.00666854
+	Iteration = 50, Sup Diff = 0.00398429
+	Iteration = 60, Sup Diff = 0.00238131
+	Iteration = 70, Sup Diff = 0.00142366
+	Iteration = 80, Sup Diff = 0.00085134
+	Iteration = 90, Sup Diff = 0.000509205
+	Iteration = 100, Sup Diff = 0.000304623
+	Iteration = 110, Sup Diff = 0.000182265
+	Iteration = 120, Sup Diff = 0.00010907
+	Iteration = 130, Sup Diff = 6.52764e-05
+	Iteration = 140, Sup Diff = 3.90711e-05
+	Iteration = 150, Sup Diff = 2.33881e-05
+	Iteration = 160, Sup Diff = 1.40086e-05
+	Iteration = 170, Sup Diff = 8.39132e-06
+	Iteration = 180, Sup Diff = 5.02647e-06
+	Iteration = 190, Sup Diff = 3.0109e-06
```

Comment:
That's a pretty long log, you could use [...] in the middle, I'm not 100% sure the whole trace is interesting for the reader.